secureli/modules/shared/models/scan.py

```diff
@@ -21,6 +21,7 @@ class ScanFailure(pydantic.BaseModel):
     repo: str
     id: str
     file: str
+    exitCode: str
```

Comment:
Q: can we type this as an `ExitCode` instead of a `string`?

Reply:
@doug-szeto-slalom The is a definition class,  pydantic is like Typescript in javascript where you define the model data types of the variables to force proper typing.